test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/schema.py

```diff
@@ -448,6 +448,7 @@ def doc_schema(module_name, for_collection=False, deprecated_module=False):
         'options': Any(None, *list_dict_option_schema(for_collection)),
         'extends_documentation_fragment': Any(list_string_types, *string_types),
         'version_added_collection': collection_name,
+        'orphaned': Any(None, bool),
```

Comment:
Shouldn't this be:
```suggestion
        'orphaned': bool,
```
Instead of specifying `None`, on can simlpy leave `orphaned` away or specify `false`.

Reply:
done, thanks!